### PR TITLE
update parl version requirement

### PR DESCRIPTION
**Pull request by TomorrowIsAnOtherDay**

### CODE REVIEW

File: examples/DQN_variant/README.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ Performance of **Dueling DQN** on various environments:
 ## How to use
 ### Dependencies:
 + [paddlepaddle>=2.0.0](https://github.com/PaddlePaddle/Paddle)
-+ [parl](https://github.com/PaddlePaddle/PARL)
++ [parl>=2.0.0](https://github.com/PaddlePaddle/PARL)
```
Comment (rical730) [inline examples/DQN_variant/README.md]: parl>=2.0.1 is required in DQN_variant.

File: examples/ES/README.md [hunk 2]
```diff
@@ -15,7 +15,7 @@ Please see [here](https://github.com/openai/mujoco-py) to know more about Mujoco
 ## How to use
 ### Dependencies
 + [paddlepaddle>=2.0.0](https://github.com/PaddlePaddle/Paddle)
-+ [parl>=1.4](https://github.com/PaddlePaddle/PARL)
++ [parl>=2.0.0](https://github.com/PaddlePaddle/PARL)
```
Comment (rical730) [inline examples/ES/README.md:18]: parl>=2.0.2？

Comment (TomorrowIsAnOtherDay) [reply examples/ES/README.md:18]: ES does not develop the algorithm in `parl.algorithm`.

File: examples/DQN_variant/README.md (hunk 1 above)
Comment (TomorrowIsAnOtherDay) [reply examples/DQN_variant/README.md]: fixed.

**Review by zenghsh3 — APPROVED** [review]

**Pull request merged**